docs/libpod.conf.5.md

```diff
@@ -65,13 +65,13 @@ libpod to manage containers.
 **cni_plugin_dir**=""
   Directories where CNI plugin binaries may be located
 
-**pause_image** = ""
-  Pause container image name for pod pause containers.  When running a pod, we
+**infra_image** = ""
+  Ifra (pause) container image name for pod infra containers.  When running a pod, we
```

Comment:
s/Ifra/Infra/

Reply:
On second thought, might be better to do s/Ifra/A/

Reply:
Fixed to say Infra